testnets/cosmosicsprovidertestnet/chain.json

```diff
@@ -33,23 +33,20 @@
   },
   "codebase": {
     "git_repo": "https://github.com/cosmos/gaia",
-    "recommended_version": "v18.0.0-rc3",
+    "recommended_version": "v20.0.0-rc0",
     "compatible_versions": [
-      "v18.0.0-rc3"
+      "v20.0.0-rc0"
     ],
-    "cosmos_sdk_version": "v0.47.16-ics-lsm",
-    "ibc_go_version": "v7.6.0",
+    "cosmos_sdk_version": "v0.50.9-lsm",
+    "ibc_go_version": "v8.5.1",
```

Comment:
the sdk property and the ibc property will need their version numbers updated as well